portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase

```diff
@@ -546,46 +519,36 @@ definition {
 	@description = "This is a use case for LPS-71236 and LPS-71307. View web content"
 	@priority = "5"
 	test ViewWebContent {
-		task ("Add CP") {
-			Navigator.openURL();
-
-			ProductMenu.gotoPortlet(
-				category = "Content &amp; Data",
-				portlet = "Web Content");
-
-			WebContentNavigator.gotoAddCP();
-
-			WebContent.addCP(
-				webContentContent = "WC WebContent Content",
-				webContentTitle = '''<script>alert(123);</script>''');
-
-			PortletEntry.publish();
+		task ("Add web content") {
+			JSONWebcontent.addWebContent(
+				content = "WC WebContent Content",
+				groupName = "Guest",
+				title = '''<script>alert(123);</script>''');
 		}
 
-		task ("Assert correct title") {
-			WebContent.viewTitle(
-				webContentTitle = '''<script>alert(123);</script>''',
-				webContentTitleEscapeCharacter = '''&lt;script&gt;alert(123);&lt;/script&gt;''');
+		task ("Add public layout") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Web Content Display Page");
 		}
 
-		task ("Add WC display widget") {
-			Navigator.openURL();
-
-			Click(locator1 = "Icon#EDIT_PENCIL");
+		task ("Add WC Display widget to public layout") {
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Guest",
+				layoutName = "Web Content Display Page",
+				widgetName = "Web Content Display");
+		}
 
-			PageEditor.addWidget(
-				portletName = "Web Content Display",
-				targetFragmentName = "Container");
+		task ("Select web content and assert no XSS pop up") {
+			Navigator.openSpecificURL(url = "http://localhost:8080/web-content-display-page");
```

Comment:
Please use the PortalURL param here also due, we don't know what Portal host URL will be:
See test results: https://testray.liferay.com/reports/production/logs/2021-11/test-1-11/test-portal-acceptance-pullrequest(master)/9217/functional-tomcat90-mysql57-jdk8/6/23/LocalFile.XSS_ViewWebContent/summary.html.gz